milli/src/search/new/matches/matching_words.rs

```diff
@@ -149,7 +152,7 @@ pub type WordId = u16;
 /// In these cases we need to match consecutively several tokens to consider that the match is full.
 #[derive(Debug, PartialEq)]
 pub enum MatchType<'a> {
-    Full { char_len: usize, ids: &'a RangeInclusive<WordId> },
+    Full { char_count: usize, byte_len: usize, ids: &'a RangeInclusive<WordId> },
```

Comment:
I added `byte_len` here so I can use it on prefix matches, but it seems that it doesn't work as it should, `token.byte_end - token.byte_start` still returns the normalized lemma length, at least it seemed so when I was testing it. That seems like a bug to me.

Reply:
Indeed yes, I will check in charabia if a normalizer didn't change the lemma content 🤔